compiler/noirc_frontend/src/hir/type_check/mod.rs

```diff
@@ -458,7 +458,7 @@ mod test {
         }
 
         fn local_module_id(&self) -> LocalModuleId {
-            LocalModuleId(arena::Index::from_raw_parts(0, 0))
+            LocalModuleId(arena::Index(0, generational_arena::Index::from_raw_parts(0, 0)))
```

Comment:
Looks like this wasn't updated. It will cause frontend tests to fail to compile I think